releasenotes/notes/fix-complex-param-binding-0fcad20135aa7382.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Fixed an issue causing an error when binding a complex parameter value to an operator's
+    coefficient. Casts to ``float`` in :class:`~qiskit.opflow.primitive_ops.PrimitiveOp`
+    and :class:`~qiskit.extensions.HamiltonianGate` were generalized to casts to ``complex``.
+    Fixes `#6889 <https://github.com/Qiskit/qiskit-terra/issues/6976>`__.
```

Comment:
The number 6889 and the issue number in the link 6976 do not match - I assume the former is incorrect as the link refers to the issue this PR is said to solver

Reply:
Thanks, a copy-paste error.